book/chrome.md

```diff
@@ -683,6 +683,18 @@ class Chrome:
         self.tabbar_bottom = self.font_height + 2*self.padding
 ```
 
+We will store rectangles representing the size of various elements in the
+browser chrome. For that, a new `Rect` class will be convenient:
+
+``` {.python}
+class Rect:
+    def __init__(self, left, top, right, bottom):
+        self.left = left
+        self.top = top
+        self.right = right
+        self.bottom = bottom
+```
```

Comment:
Do you want to also ask people to change `DrawRect` to take in a `Rect` argument? We could even ask the reader to replace the `top`/`left`/`right`/`bottom` fields on drawing commands like `DrawText` and `DrawLine`, which we use for clipping, with a per-drawing-command `rect` field. I think this chapter has typically been easy (and the hardest part was visualizing coordinnates, which we've eliminated with auto-sizing) so I'd rather add here to remove from Ch11.